DynamicActiveQuery.php

```diff
@@ -155,12 +202,95 @@ public function createCommand($db = null)
             % (`?) \{
                 ( [a-z_\x7f-\xff][a-z0-9_\x7f-\xff]* (?: \. [^.|\s]+)* )
                 (?:  \| (binary (?:\(\d+\))? | char (?:\(\d+\))? | time (?:\(\d+\))? | datetime (?:\(\d+\))? | date
-                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?(?:,\d\d?)?\))?
-                        | int(eger)? | (?:un)? signed (?:\int(eger)?)?)  )?
+                        | decimal (?:\(\d\d?(?:,\d\d?)?\))?  | double (?:\(\d\d?,\d\d?\))?
+                        | int(eger)? | (?:un)? signed(?:\s\int(eger)?)?)  )?
```

Comment:
because, as i understand, type is `(un)?signed int(eger)` - so there should be space symbol, isn't it?